its/projects/SharedFilesTest/Class1.cs

```diff
@@ -1,6 +1,4 @@
-﻿using System;
-
-public class Class1
+﻿public class Class1
```

Comment:
Because it was creating another issue in the file (unused `using`), so it was either removing it here, or change the assertion [here](https://github.com/SonarSource/sonar-dotnet/blob/052f8b4d83969af5518f9449665c7fcb58c0dad5/its/src/test/java/com/sonar/it/csharp/SharedFilesTest.java#L76).
I can put it back and change the assertion if you prefer

Reply:
no, that's ok